CONTRIBUTORS.rst

```diff
@@ -87,6 +87,7 @@ Listed in alphabetical order.
   Chris Pappalardo         `@ChrisPappalardo`_
   Christopher Clarke       `@chrisdev`_
   Cole Mackenzie           `@cmackenzie1`_
+  Cole Maclean             `@cole`_                      @cole
```

Comment:
Just a note that this is a broken link in rst. To fix it, you should add the target as the others further down the file.

Reply:
Oops, of course. Thanks!